### ISSUE TITLE
Undefined Variable: tagfield

### ISSUE THREAD

**Issue by rgreenblatt** [liuchengxu/vista.vim#60]
**Describe the bug**
Vista spams undefined variable `tagfield` when in certain files. Specifically, I've been able to replicate the problem in the source code of some vim plugins. I have isolated the problem to [this](https://github.com/liuchengxu/vista.vim/blob/master/autoload/vista/parser/ctags.vim#L45) line. I could be missing something, but as far as I can tell, `tagfield` will always be undefined in that line. Changing `tagfield` to `a:tagfields[0]`, which I think is the intention, completely removes the problem. I can make a PR if that would be helpful.

**Environment:**
- OS: Ubuntu 18.10
- Vim/Neovim version: NVIM v0.4.0-dev, Build type: RelWithDebInfo
- This plugin version: 3da62ec98105f9d6686c28bd2d40a927cd016d74
- It may be relevant that I am using coc.nvim with [coc-neco](https://github.com/neoclide/coc-neco)

**Vista info**

```
Current FileType: vim
Avaliable Executives: ['coc', 'ctags']
Global Variables:
let g:vista#executives = ['coc', 'ctags', 'lcn', 'vim_lsp']
let g:vista_fzf_preview = ['right:50%']
```

**Steps to reproduce given the above info**
I think this bug is pretty straight forward, but if a minimal init.vim/vimrc which reproduces the problem would be helpful let me know.

**Comment by liuchengxu** [comment]
Can you provide the source file you were running Vista on?

**Comment by liuchengxu** [comment]
Fixed by #61
